dandi/metadata.py

```diff
@@ -83,3 +88,62 @@ def get_metadata(path):
     meta["nd_types"] = get_neurodata_types(path)
 
     return meta
+
+
+def nwb_to_json(fpath, out_fpath):
+    io = NWBHDF5IO(fpath, 'r')
+    file = File(fpath)
+    io.read()
+    root_builder = io.get_builder(file['general']).parent
+    out_dict = group_builder_to_json(root_builder)
+    return json.dump(out_dict, out_fpath)
+
+
+def print_dataset(dataset):
+    return str(dataset)  # may want to change
+
+
+def dataset_builder_to_json(dataset_builder):
```

Comment:
it doesn't return "JSON" (there is no such "thing") -- returns `dict`.  
Also it is nwb specific -- I feel that this and other (all?) nwb specific functions should go into `dandi/pynwb_utils.py` (it is ok to "interface" hdmf there too ;))